cribl_stream/metadata.csv

```diff
@@ -0,0 +1,52 @@
+cribl.logstream.sourcetype.out_events,counter,,,,Outbound events by sourcetype,,,,,,,,,,,,,,,,,,
+cribl.logstream.total.in_bytes,counter,,,,Total inbound bytes,,,,,,,,,,,,,,,,,,
+cribl.logstream.total.in_events,counter,,,,Total inbound events,,,,,,,,,,,,,,,,,,
+cribl.logstream.total.out_bytes,counter,,,,Total outbound bytes,,,,,,,,,,,,,,,,,,
+cribl.logstream.total.out_events,counter,,,,Total outbound events,,,,,,,,,,,,,,,,,,
+cribl.logstream.total.dropped_events,counter,,,,Dropped events total,,,,,,,,,,,,,,,,,,
+cribl.logstream.health.inputs,gauge,,,,Healthy inputs,,,,,,,,,,,,,,,,,,
+cribl.logstream.health.outputs,gauge,,,,Healthy outputs,,,,,,,,,,,,,,,,,,
+cribl.logstream.system.load_avg,gauge,,,,Load Average,,,,,,,,,,,,,,,,,,
+cribl.logstream.system.free_mem,gauge,,,,Free memory,,,,,,,,,,,,,,,,,,
+cribl.logstream.system.disk_used,gauge,,,,Disk usage,,,,,,,,,,,,,,,,,,
+cribl.logstream.system.cpu_perc,gauge,,,,CPU percentage usage,,,,,,,,,,,,,,,,,,
+cribl.logstream.system.mem_rss,gauge,,,,Ram usage,,,,,,,,,,,,,,,,,,
+cribl.logstream.total.activeCxn,gauge,,,,Total active inbound TCP connections,,,,,,,,,,,,,,,,,,
+cribl.logstream.pipe.in_events ,counter,,,,Inbound events per Pipeline,,,,,,,,,,,,,,,,,,
+cribl.logstream.pipe.out_events ,counter,,,,Outbound events per Pipeline,,,,,,,,,,,,,,,,,,
+cribl.logstream.pipe.dropped_events ,counter,,,,Dropped events per Pipeline,,,,,,,,,,,,,,,,,,
+cribl.logstream.metrics_pool.num_metrics ,gauge,,,,The total number of unique  metrics that have been allocated into memory.,,,,,,,,,,,,,,,,,,
+cribl.logstream.collector_cache.size ,counter,,,,"Each Collector function  (default/cribl/collectors/<collector>/index.js) isloaded/initializedonly once per  job, and  then cached. This metric  representsthe  current size ofthis cache.",,,,,,,,,,,,,,,,,,
+cribl.logstream.cluster.metrics.sender.inflight ,gauge,,,,"Number of metric  packets currently being sent from a Worker/Edge Node Process to the API Process, via IPC (interprocess   communication).",,,,,,,,,,,,,,,,,,
+cribl.logstream.backpressure.outputs ,counter,,,,"Destinations experiencing backpressure, causing events to be either blocked or dropped.",,,,,,,,,,,,,,,,,,
+cribl.logstream.blocked.outputs ,counter,,,,Blocked Destinations. (This metric is more restrictive than the one listed just above.),,,,,,,,,,,,,,,,,,
+cribl.logstream.pq.queue_size ,gauge,,,,"Current queue size, per Destination, per Worker/Edge NodeProcess.",,,,,,,,,,,,,,,,,,
+cribl.logstream.host.in_bytes ,counter,,,,Inbound bytes from a given host (host is a characteristic of the data).,,,,,,,,,,,,,,,,,,
+cribl.logstream.host.in_events ,counter,,,,Inbound events from a given host (host is a characteristic of the data).,,,,,,,,,,,,,,,,,,
+cribl.logstream.host.out_bytes ,counter,,,,Outbound bytes from a given host (host is a characteristic of the data).,,,,,,,,,,,,,,,,,,
+cribl.logstream.host.out_events ,counter,,,,Outbound events from a given host (host is a characteristic of the data).,,,,,,,,,,,,,,,,,,
```

Comment:
Just to clarify, does `characteristic of the data` mean a [tag](https://docs.datadoghq.com/getting_started/tagging/#overview) in this context?

Reply:
This can be a tag.

Reply:
Thanks for confirming that. Could you update the metric descriptions which currently mention `host is a characteristic of the data` to specify the tag being used? For example, is it a `<Key>:<Value>` or `<Value>` format tag? My goal is to make sure that customers and support can reference the documentation to know how to search for these hosts.

Reply:
No, actually this does not relate to a tag. This is just saying that the metric is related to a specific host sending data through Cribl Stream. The section (host is a characteristic of the data) can be removed if it's complicating the event.